AsyncDisplayKit/ASPagerNode.h

```diff
@@ -120,6 +120,24 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (NSInteger)indexOfPageWithNode:(ASCellNode *)node;
 
+/**
+ * Tells the pager node to allow its view controller to automatically adjust its content insets.
+ *
+ * @see UIViewController.automaticallyAdjustsScrollViewInsets
+ *
+ * @discussion ASPagerNode should usually not have its content insets automatically adjusted
+ * because it scrolls horizontally, and flow layout will log errors because the pages
+ * do not fit between the top & bottom insets of the collection view.
+ *
+ * The default value is NO, which means that ASPagerNode expects that its view controller will
+ * have automaticallyAdjustsScrollViewInsets=NO.
+ *
+ * If this property is NO, but your view controller has automaticallyAdjustsScrollViewInsets=YES,
+ * the pager node will set the property to NO and log a warning message. In the future,
```

Comment:
This isn't 100% clear to me. It will set the property on the view controller to NO? Will it **leave** the pageNode property NO? Or, will it actually set the pagerNode property to YES?